content/en/docs/concepts/security/multi-tenancy.md

```diff
@@ -514,4 +514,4 @@ project provides an implementation of virtual control planes.
 #### Other implementations
 
 * [Kamaji](https://github.com/clastix/kamaji)
-* [vcluster](https://github.com/loft-sh/vcluster)
\ No newline at end of file
+* [vcluster](https://github.com/loft-sh/vcluster)
```

Comment:
@mikejoh, does this new addition provide any changes to the original content.

Reply:
No, i missed this, sorry. I was editing the page via the "Edit this page", this opens the file in the GitHub web based editor. Somehow that edit added another change.

When i diff using `git` locally i can see the following:
```
 * [Kamaji](https://github.com/clastix/kamaji)
-* [vcluster](https://github.com/loft-sh/vcluster)
\ No newline at end of file
+* [vcluster](https://github.com/loft-sh/vcluster)
```
It complains that there's no newline at end of file. Maybe the web based editor added something when i saved.

I could try to redo the commit and push again by checking out the file from `main` and only change what i intended to.